### PR TITLE
adding helm-mutator

### PR DESCRIPTION
**Pull request by TimofeiShorokhov**

### CODE REVIEW

**Comment by Denislite** [comment]
Same commit names. Combine them into one.

File: mutator-helm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: Mutator-HelmChart
+description: Helm for mutator
+type: application
+version: 0.1.0
+appVersion: "1.0.0"
```
Comment (vangithub1992) [inline mutator-helm/Chart.yaml:6]: I'm advising to use an empty line at the EOF

File: mutator-helm/templates/webhook-configuration.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: {{ .Values.appName }}
+webhooks:
```
Comment (vangithub1992) [inline mutator-helm/templates/webhook-configuration.yaml:5]: maybe we can describe YAML fields for better understanding?

**Review by vangithub1992 — APPROVED** [review]

File: mutator-helm/templates/webhook-configuration.yaml (hunk 2 above)
Comment (TimofeiShorokhov) [reply mutator-helm/templates/webhook-configuration.yaml:5]: Added description of fields in values.yaml

File: mutator-helm/Chart.yaml (hunk 1 above)
Comment (TimofeiShorokhov) [reply mutator-helm/Chart.yaml]: fixed

**Review by Kaborda-Irina — APPROVED** [review]

**Review by Denislite — APPROVED** [review]
LGTM

**Review by VadzimBelski-ScienceSoft — APPROVED** [review]

**Pull request merged**